source/model.js

```diff
@@ -91,57 +107,61 @@ function ComputerPlayer(name, symbol, field){
 ComputerPlayer.prototype = Object.create(Player.prototype);
 ComputerPlayer.prototype.constructor = ComputerPlayer;
 
-function Game(humanPlayer, firstPlayerName, secondPlayerName){
+function Iterator(array) {
+    let index = 0;
+    this.getCurrent = function(){
+        return array[index];
+    };
+    this.moveNext = function(){
+        index = index == array.length-1 ? 0 : ++index;
+    }
+}
+
+function Game(players){
     let self = this;
-    let currentPlayerIndex = 0;
+    this.field = singletonField.getInstance();
+    this.iterator = new Iterator(players);
```

Comment:
Can we make the iterator private too? We can reveal something like a "currentPlayerName" field in the Game instance instead. If you do, please place the aforementioned variable right above the currentStatus.